mycompany-seeq-link-connector-myconnector/build.gradle.kts

```diff
@@ -7,11 +7,9 @@ group = "com.mycompany.seeq.link.connector"
 version = "0.1.0-SNAPSHOT"
 
 dependencies {
-    compileOnly("com.seeq.link:seeq-link-sdk:54.3.0-v202201281139")
-    //    compileOnly("com.seeq.link:seeq-link-sdk:55.1.0-v202202032202-BETA")
+    compileOnly("com.seeq.link:seeq-link-sdk:59.1.1-v202301172059")
```

Comment:
Good idea. I always changed the release branches but it is nice to also update main.